src/test/java/com/artipie/cache/CachedUsersTest.java

```diff
@@ -67,14 +85,18 @@ void failsToGetUserWhenCacheContainsAnotherUser() {
     @Test
     void getsUserFromCache() {
         final String user = "super_user";
-        final CachedUsers target = new CachedUsers();
-        target.invalidateAll();
+        final CachedUsers target = new CachedUsers(this.cache);
         target.user(
             user, "", (usr, pass) -> Optional.of(new Authentication.User(user))
         );
         MatcherAssert.assertThat(
             target.user(user, "", (usr, pass) -> Optional.empty()).isPresent(),
             new IsEqual<>(true)
         );
+        MatcherAssert.assertThat(
+            "Size of cache differed from 1",
+            this.cache.size(),
+            new IsEqual<>(1L)
```

Comment:
It's a minor issue, but you can use [Matchers.hasSize](http://hamcrest.org/JavaHamcrest/javadoc/2.2/org/hamcrest/Matchers.html#hasSize-int-) next time.